### PR TITLE
AmpTask1786_Integrate_20220426

### PR DESCRIPTION
**Pull request by gpsaggese**

### CODE REVIEW

File: dataflow/notebooks/gallery_dataflow_example.py [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+    full_symbol_list = df["full_symbol"].unique()
+    for cc in full_symbol_list:
+        # DataFrame with a specific `full_symbol`
+        cc_df = df[df["full_symbol"] == cc]
+        # Resample OHLCV data inside `full_symbol`-specific DataFrame.
+        resampled_cc_df = cfinresa.resample_ohlcv_bars(
+            cc_df, rule=resampling_freq
+        )
+        # Attach VWAP, TWAP.
+        resampled_cc_df[["vwap", "twap"]] = cfinresa.compute_twap_vwap(
```
Comment (gitpaulsmith) [inline dataflow/notebooks/gallery_dataflow_example.py:153]: This function is to be deprecated.

File: dataflow/system/example_pipeline1_system_runner.py [hunk 2]
```diff
@@ -93,12 +191,40 @@ def get_portfolio(
...
 
-# #############################################################################
+class Example1_Database_ForecastSystem(
+    dtfsysyrun.SystemWithSimulatedOmsRunner,
+    Example1_SystemRunner
+    # Example1_ForecastSystem
+):
 
+    def get_dag(
+            self,
```
Comment (gitpaulsmith) [inline dataflow/system/example_pipeline1_system_runner.py:201]: Fix indentation.

File: dataflow/system/system_runner.py [hunk 3]
```diff
@@ -19,10 +19,92 @@
...
+
+    @abc.abstractmethod
+    def get_dag_config(
+        self,
+        prediction_col: str,
+        volatility_col: str,
+        returns_col: str,
+        timedelta: pd.Timedelta,
+        asset_id_col: str,
+        *spread_col: Optional[str],
```
Comment (gitpaulsmith) [inline dataflow/system/system_runner.py:71]: Typo?

File: dataflow/system/system_runner.py [hunk 4]
```diff
@@ -19,10 +19,92 @@
...
+    """
+
+    @abc.abstractmethod
+    def get_market_data(
+        self, event_loop: asyncio.AbstractEventLoop
+    ) -> mdata.MarketData:
+        ...
+
+    @abc.abstractmethod
+    def get_dag_config(
```
Comment (gitpaulsmith) [inline dataflow/system/system_runner.py:64]: I would consider adding a price column. How are these being used? They shouldn't be needed to configure or run the DAG.

File: dataflow/system/test/outcomes/Test_Example1_ForecastSystem.test1/output/test.txt [hunk 5]
```diff
@@ -0,0 +1,287 @@
...
+       #   Column                 Non-Null Count  Dtype                           
+      ---  ------                 --------------  -----                           
+       0   (close, 101)           5 non-null      float64                         
+       1   (feature1, 101)        5 non-null      float64                         
+       2   (start_datetime, 101)  5 non-null      datetime64[ns, America/New_York]
+       3   (timestamp_db, 101)    5 non-null      datetime64[ns, America/New_York]
+       4   (volume, 101)          5 non-null      int64                           
+      dtypes: datetime64[ns, America/New_York](2), float64(2), int64(1)
+payload: None
+class: ResultBundle
```
Comment (gitpaulsmith) [inline dataflow/system/test/outcomes/Test_Example1_ForecastSystem.test1/output/test.txt:287]: Let's revisit this clean-up soon!

File: dataflow/notebooks/gallery_dataflow_example.py (hunk 1 above)
Comment (cryptomtc) [reply dataflow/notebooks/gallery_dataflow_example.py:153]: Yes, it was just a placeholder. I'll add a TODO and update in a follow up.

File: dataflow/system/system_runner.py (hunk 3 above)
Comment (cryptomtc) [reply dataflow/system/system_runner.py:71]: yes, fixed

File: dataflow/system/system_runner.py (hunk 4 above)
Comment (cryptomtc) [reply dataflow/system/system_runner.py:64]: image

**Pull request merged**